TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/Http.rst

```diff
@@ -226,15 +235,28 @@ The second method, ``createFromEnvironment()``, take the environment provided by
 functions into account. It creates a ``Request`` instance which reflects the current HTTP request received from the web
 server. This method is best used if you need a ``Request`` object with all properties set according to the current
 server environment and incoming HTTP request.
+Note though, that you should not expect this ``Request`` to match the current ``Request``, since the latter will still
+have been affected by some ``HTTP Components``. If you need the **current** Request, get it from the ``RequestHandler`` instead.
 
 Creating an ActionRequest
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In order to dispatch a request to a controller, you need an ``ActionRequest``.
 Such a request is always bound to an ``Http\Request``::
 
-	$httpRequest = Request::createFromEnvironment();
-	$actionRequest = new ActionRequest($httpRequest);
+	/**
+	 * @var Bootstrap
```

Comment:
I'd suggest to in include the `use` statements instead as we encourage to import namespace for readabilitys sake in our CGL